src/darkgraylib/tests/test_git.py

```diff
@@ -453,3 +453,23 @@ def test_git_get_root_not_found(tmp_path, path):
     root = git.git_get_root(tmp_path / path)
 
     assert root is None
+
+
+def test_git_output_in_english(git_repo, monkeypatch):
+    """Git output is in English despite German language environment."""
+    # Set up German language environment
+    german_env = {
+        "LC_ALL": "de_DE.UTF-8",
+        "LC_CTYPE": "de_DE.UTF-8",
+        "LC_MESSAGES": "de_DE.UTF-8",
+        "LANG": "de_DE.UTF-8",
+    }
```

Comment:
Setting `LC_ALL` should be enough afaik, as this overrides all the other `LC_*` settings as well as `LANG`

Reply:
Here I actually want to test that `LC_ALL=C.UTF-8` overrides the language even if all these other environment variables still specify a different language.